apps/prairielearn/src/pages/administratorJobSequence/administratorJobSequence.html.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function administratorJobSequence({
```

Comment:
By convention, page "components" like this use PascalCase:

```suggestion
export function AdministratorJobSequence({
```

You'll have to make this change where this is imported as well.